### ISSUE TITLE
Wrong view.sliderHeader when switching to map page right after app launch

### ISSUE THREAD

**Issue by opeeters** [irceline/aq-mobile-be#160]
When going directly to the map page after launching the app after previous launches you land in the BelAQI-tab, which is perfect, but, the `view.sliderHeader` is `annual mean (long term exposure)`. 

This is corrected when switching to a different pollutant tab.

For the BelAQI-tab the `view.sliderHeader` should however also rather be "BelAQI (short term exposure)" instead of "hourly mean (short term exposure)". This could be seen as an enhancement for milestone 1.1 if this is a bit to involved. The bug part needs to be fixed for 1.0.

**Comment by SpeckiJ** [comment]
This should both be fixed as of https://github.com/irceline/aq-mobile-be/commit/30db164d863b488ef720c3149e21d214551b720b

I am relying on the fact that all translations have a round bracket to denote the end of the "annual mean" part and start the "short-term exposure" part in the respective languages. If this should ever change this code needs to be adjusted accordingly.

**Comment by opeeters** [comment]
@SpeckiJ Thanks for the fast fix!

**Comment by opeeters** [comment]
@SpeckiJ 
This still remains an issue now, only after app launch, when going via the `short-term` or `long-term` panel.

**Comment by SpeckiJ** [comment]
Unfortunately I cannot spot the issue, all seems fine when I launch the App. @opeeters Can you describe the unwanted behaviour in more detail please?

**Comment by opeeters** [comment]
@SpeckiJ 
Hmm, one of these, sometime yes, sometimes no issues..
Steps to reproduce:
1) add one user location (e.g. Leuven) and activate "current location" (with "current location" deactivated and two user locations you don't have the issue)
2) relaunch the app
3) via any of the "short-term" or "long-term" icons you land in BelAQI (short term exposure)

**Comment by opeeters** [comment]
might be related to speed with which current location is established by the device..

**Comment by SpeckiJ** [comment]
Thanks for the detailed description :+1:
--> With this description I was able to recreate and fix the error (Explanation: When using current location a dummy location is used when initiating the Map and the real location only supplied later on. As the navigation to the correct map from a panel was done via a timeout (300ms), the location was not always available yet and the navigation therefore failed).

I have refactored part of the code in https://github.com/irceline/aq-mobile-be/commit/71ecdb859930d638653a68a261dd26339f5dd1be and the issue seems to be fixed now.

Sorry for the late reply but the bug was more complicated than initally estimated.
